### ISSUE TITLE
Add fast_hocr_only mode

### ISSUE THREAD

**Issue by Hinote-github** [phil-el/phetools#3]
Actually, we need a mode for fast_hocr where it does not fall into slow_hocr even if it cannot produce data from the existing OCR layer.

The tesseract software used at toollabs runs bad on Russian texts. That was the reason why the OCR gadget has not been enabled at ru.ws several years ago and is not enabled in the meantime. It (tesseract) was bad several years ago and still produces text layer with inacceptable quality...

In the same time, we would like to utilize the fast_hocr functionality -- it works brilliant on existing text layers splitting the existing texts into paragraphes...

So, as an implementation idea, you could add the '&mode=fast_hocr_only' or something like that to the URL fields recognized by the hocr web front-end and call only fast_hocr() in this case with no attempts to call slow_hocr().

Thanks in advance.

**Comment by phil-el** [comment]
The code path is a bit too complicate to add  new mode, rather to add a new parameter I did https://github.com/phil-el/phetools/commit/ae6f5f08a6681a734a2b83d903ad11fd7f3183a5 to disable fast ocr for the whole ru.ws, I'm unsure if it's what you need. One problem not solved : I've on the server side many hocr already done for ru.ws but I didn't keep a track of which was produced by the slow or the fast path... All pdf are in this case.

**Comment by Hinote-github** [comment]
Yeah, the patch you implemented may be used for a while as a quick&dirty solution for our ru ws (need to be tested, indeed).

However, I'd like to not hardcode this dependency for lang=ru since there may be two different buttons in the toolbar, one with fast_hocr ony functionality and another one with the same functionality falling to slow_ocr as for other languages... I guess I'll try to implement the new mode parameter to the web front-end by myself and propose the changes to you for review...

And, yes, you'll better distinguish between fast_ocr and slow_ocr queries in the tracks somehow for better failure diagnostics. For example, you could modify the cmd field accodingly (i.e. "hocr.py fast_hocr()" and "hocr.py slow_hocr()" instead of simply using "hocr.py"...

Could you also please clear all caches for lang=ru in order to be sure that all they are created by fast_ocr, in the new way. I'll help testing latest changes made for ru ws...

(If the caches are cleaned up after, say, a week or month after creation?)
